### PR TITLE
feat: add new color system to action toggle, action trigger, badge and breadcrumb

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Updated `<ActionToggle/>`, `<ActionTrigger/>`, `<Badge/>` and `<Breadcrumb/>` styles to use new color system. 

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by chrisdholt** [comment]
@eljefe223 can we validate that the action toggle changes work with the v4 appearance change that's in PR right now?

File: packages/fast-components-styles-msft/src/action-trigger/index.ts [hunk 1]
```diff
@@ -39,67 +39,10 @@ export const actionTriggerButtonOverrides: ComponentStyles<
...
-    const secondaryDisabledBackgroundColor: string = disabledContrast(
-        contrastScale,
-        secondaryBackgroundColor,
-        backgroundColor
-    );
-    const secondaryDisabledColor: string = disabledContrast(
-        contrastScale,
-        foregroundColor,
-        designSystem.brandColor
-    );
```
Comment (nicholasrice) [inline packages/fast-components-styles-msft/src/action-trigger/index.ts]: all that reeeeedddddd.... love it

File: packages/fast-components-styles-msft/src/action-trigger/index.ts [hunk 2]
```diff
@@ -39,67 +39,10 @@ export const actionTriggerButtonOverrides: ComponentStyles<
...
-    const backgroundColor: string = designSystem.backgroundColor;
-    const brandColor: string = designSystem.brandColor;
     const direction: Direction = designSystem.direction;
-    const scaledNormalContrast: ContrastFunction = curry(normalContrast)(contrastScale);
-    const scaledEnsureNormalContrast: ContrastFunction = curry(ensureNormalContrast)(
-        contrastScale
-    );
-
-    // Define secondary button colors
     const color: string = "white";
```
Comment (nicholasrice) [inline packages/fast-components-styles-msft/src/action-trigger/index.ts]: I don't think you still need this `color` var

File: packages/fast-components-styles-msft/src/breadcrumb/index.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ const styles: ComponentStyles<BreadcrumbClassNameContract, DesignSystem> = (
             ...typographyPattern.rest,
```
Comment (nicholasrice) [inline packages/fast-components-styles-msft/src/breadcrumb/index.ts]: are `designSystem` and `direction` variables above being used? if not, can you remove them?

Comment (eljefe223) [reply packages/fast-components-styles-msft/src/breadcrumb/index.ts]: yes for applyLocalizedProperty

**Pull request merged**